### PR TITLE
docs(angular.md): update to enable NgModule to contain Non-Angular elements

### PR DESCRIPTION
**Pull request by spas906**
Update Docs to enable NgModule to contain Non-Angular elements

# Description

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
Thanks @spas906 I'm sure the Angular community will really appreciate this bit of info 😄

File: packages/web-components/fast-foundation/docs/integrations/angular.md [hunk 1]
```diff
@@ -87,6 +87,17 @@ export class AppComponent {
 }
 ```
 
+
+To allow NgModule to contain Non-Angular elements named, add  the code in your `app/app.module.ts` file contents with this:
```
Comment (awentzel) [inline packages/web-components/fast-foundation/docs/integrations/angular.md:91]: I think you could just say "Allows NgModule..." and there is a double space in ", add  the code".

File: packages/web-components/fast-foundation/docs/integrations/angular.md [hunk 2]
```diff
@@ -87,6 +87,17 @@ export class AppComponent {
...
+
+```ts 
+import {CUSTOM_ELEMENTS_SCHEMA  } from '@angular/core';
```
Comment (awentzel) [inline packages/web-components/fast-foundation/docs/integrations/angular.md:94]: the spaces are off inside the `{..}` brackets.  Prettier may catch, this though I'm not 100% positive

File: packages/web-components/fast-foundation/docs/integrations/angular.md (hunk 1 above)
Comment (EisenbergEffect) [inline packages/web-components/fast-foundation/docs/integrations/angular.md:91]:
```suggestion
To allow an NgModule to contain Non-Angular element names, add the following code in your `app/app.module.ts` file:
```

File: packages/web-components/fast-foundation/docs/integrations/angular.md (hunk 2 above)
Comment (EisenbergEffect) [inline packages/web-components/fast-foundation/docs/integrations/angular.md:94]:
```suggestion
import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

**Pull request merged**